src/complex_types/repository.md

```diff
@@ -15,51 +15,52 @@ CodeGen is a powerful code generator designed for DBL. Rather than writing repet
 
 Now, imagine harnessing the detailed data definitions from Repository and feeding them into CodeGen. This integration transforms the development process. With the insights from the data dictionary, CodeGen can produce code that's tailored to the specific data structures and relationships defined in Repository. For example, if Repository defines a customer entity with specific attributes and relationships, CodeGen can automatically generate the data access layer, CRUD operations, and even user interface components for managing customer data.
 
-Furthermore, as data definitions evolve in Repository, developers can re-run CodeGen to update the corresponding parts of the application, ensuring that the software remains aligned with the latest data schema. This iterative process reduces manual errors, enhances maintainability, and ensures that the application remains data-centric.
+Furthermore, as data definitions evolve in Repository, developers can rerun CodeGen to update the corresponding parts of the application, ensuring that the software remains aligned with the latest data schema. This iterative process reduces manual errors, enhances maintainability, and ensures that the application remains data-centric.
 
-### Defining a Repository
-There's two ways to build a repository: textually using the Data Definition Language or visually using the repository program. We will focus on the data definition language in this book because it's more flexible and easier to maintain. However, the repository program is a great tool for visualizing the data dictionary and can be used to generate the data definition language.
+### Defining a repository
+There are two ways to build a repository: textually using the Synergy Data Language or visually using the repository program<!--Do we mean the Repository application"-->. We will focus on the Synergy Data Language in this book because it's more flexible and easier to maintain. However, the repository program is a great tool for visualizing the data dictionary and can be used to generate the data definition language.
 
-### Include from Repository
-The `.INCLUDE` statement is used to include a repository structure in a program. The usage of the word structure here is a bit misleading because it can produce a variety of data types, including structures, commons, records, and groups. If the repository is a Data Dictionary, think of `.INCLUDE "structure" REPOSITORY` as your language interface to the data structures stored in your data dictionary.
+### Including from a repository
+The .INCLUDE statement is used to include a repository structure in a program. The use of the word "structure" here is a bit misleading because it<!--What does "it" refer to?--> can produce a variety of data types, including structures, commons, records, and groups. Think of `.INCLUDE "structure" REPOSITORY` as your language interface to the data structures stored in your data dictionary.
```

Comment:
it refers to the word "structure" in all our .INCLUDE from repository syntax descriptions